airflow/providers/amazon/aws/triggers/glue_crawler.py

```diff
@@ -68,11 +68,8 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
                     break  # we reach this point only if the waiter met a success criteria
                 except WaiterError as error:
                     if "terminal failure" in str(error):
-                        yield TriggerEvent(
-                            {"status": "failure", "message": f"Glue Crawler creation Failed: {error}"}
-                        )
-                        break
+                        raise
```

Comment:
changing this following the discussion on slack regarding the fact that errors should be thrown in the triggers rather than returned in >=2.6